rollup-tests.config.js

```diff
@@ -70,6 +70,8 @@ export default {
               '**/test/**/*.js',
               '**/test-util/**',
               'src/components/icons/**/*.js',
+              'src/pattern-library/**/*.js',
+              'src/pattern-library/**/*.ts*',
```

Comment:
The test for this change makes use of a little utility component, `LoremIpsum`. Without updating this config, istanbul (and, later, CodeCov) will bleat because the module that `LoremIpsum` is in doesn't have tests.